Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM matchoffice/rubies:2.3.1
```

Comment:
I wasn't questioning that way to do it. It's fine, really. I was just wondering if we could save some time by pulling a baser image for it, but it probably doesn't make sense at this scale.